thrift/server.cc

```diff
@@ -232,7 +232,8 @@ thrift_server::do_accepts(int which, bool keepalive) {
     // Future is waited on indirectly in `stop()` (via `_stop_gate`).
     (void)with_gate(_stop_gate, [&, this] {
         return _listeners[which].accept().then([this, which, keepalive] (accept_result ar) {
-            auto&& [fd, addr] = ar;
+            auto&& fd = ar.connection;
+            auto&& addr = ar.remote_address;
```

Comment:
The more of these changes I read, the more confused I get... What does it mean to put an lvalue in an "auto&&"? Can't it just be written as a "auto&"?

Reply:
Yes, it can be written as `auto&`.